src/reducers/filter.js

```diff
@@ -0,0 +1,10 @@
+import * as types from '../actions/actionsType';
+
+export default (state = '', action) => {
```

Comment:
Guess you missed this, in the pathwright you're supposed to set filter default to "All".
![image](https://user-images.githubusercontent.com/31897434/83085817-e701bf00-a084-11ea-878a-ccc994bf28fe.png)

**_I trust you can make this little changes before merging._**